compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -4103,3 +4103,33 @@ declare_lint! {
     };
     report_in_external_macro
 }
+
+declare_lint! {
+    /// The `invalid_macro_export_arguments` lint detects cases where `#[macro_export]` is being used with the wrong number of arguments, or invalid ones.
+    ///
+    /// ### Example
+    ///
+    /// ```rust,compile_fail
+    /// #![deny(invalid_macro_export_arguments)]
+    ///
+    /// #[macro_export(invalid_parameter)]
+    /// macro_rules! myMacro {
+    ///    () => {
+    ///         // [...]
+    ///    }
+    /// }
+    ///
+    /// #[macro_export(too, many, items)]
+    /// ```
+    ///
+    /// {{produces}}
+    ///
+    /// ### Explanation
+    ///
+    /// The only valid list of arguments is `#[macro_export(local_inner_macros)]` or no arguments (`#[macro_export]`), you can't have multiple arguments in a macro export, or invalid ones.
+    ///
+    ///
+    pub INVALID_MACRO_EXPORT_ARGUMENTS,
+    Deny,
+    "There aren't any attributes named \"invalid_parameters\"",
```

Comment:
Why does this message reference `"invalid_parameters"`?

Reply:
Fixed